numpy/core/numeric.py

```diff
@@ -2309,9 +2312,13 @@ def isclose(a, b, rtol=1.e-5, atol=1.e-8, equal_nan=False):
 
      absolute(`a` - `b`) <= (`atol` + `rtol` * absolute(`b`))
 
-    The above equation is not symmetric in `a` and `b`, so that
-    `isclose(a, b)` might be different from `isclose(b, a)` in
-    some rare cases.
+    Unlike the builtin `math.isclose`, the above equation is not symmetric
```

Comment:
Here's another suggestion:
```
Unlike the built-in `math.isclose`, the above equation is not symmetric
in `a` and `b` -- it assumes `b` is the reference value -- so that 
`isclose(a, b)` might be different from `isclose(b, a)`. Furthermore,
the default value of `atol` is not zero, and is targeted at numbers
of order unity; especially for numbers significantly smaller than one,
it can give false positives if not adjusted appropriately.
```